Anna.Lytvyn/Lecture2/index.css

```diff
@@ -0,0 +1,141 @@
+/* nav,
+header,
+footer,
+main,
+figure,
+article,
+footer{
+  display: block;
+} */
+
+body{
+  margin: 0;
+}
+
+.page-header{
+  width: 100%;
+  position: fixed;
+}
+
+.page-header nav{
+  background-color: rgb(24, 211, 180); 
+}
+
+.page-header nav a{
+  text-decoration: none;
+  padding: 10px;
+  padding-right: 30px;
+  display: inline-block;
+  font-size: 17px;
+  font-family: 'Montserrat', sans-serif;
+  text-transform: uppercase;
+  color: rgb(32, 32, 32);
+}
+
+.page-header nav a:hover{
+  background-color: rgb(219, 241, 238);
+  color: rgb(1, 12, 2);
+}
+
+.top-image{
+  margin: 0;
+  overflow: hidden;
+}
+
+article{
+  line-height: 1.5;
+  padding-left: 17%;
+  padding-right: 17%;
+}
+
+article header h1{
+  text-align: center;
+  font-family: 'Syncopate', sans-serif;
+}
+
+article h2{
+  text-align: center;
+  font-family: 'Syncopate', sans-serif;
+}
+
+article p{
+  text-align: justify;
+  font-family: 'Montserrat', sans-serif;
+  font-size: 18px;
+  text-indent: 50px;
+  padding: 10px;
+  margin: 0;
+}
+
+.img-1{
+    position: relative;
+    padding-right: 20px;
+    height: 250px;
+    padding-right: 20px;
+    float: left;
+  margin: 0px; 
+  padding: 10px;
+  
+}
+
```

Comment:
In this case is better to split this to `image` class, and then add `float-left`, `float-right` classes and add them respectively to extend `image` class.

Reply:
This also will be more readable than abstract non-scalable img1/2